graphql_base/__manifest__.py

```diff
@@ -11,7 +11,7 @@
     "website": "https://github.com/OCA/rest-framework",
     "depends": ["base"],
     "data": ["views/graphiql.xml"],
-    "external_dependencies": {"python": ["graphene", "graphql_server"]},
+    "external_dependencies": {"python": ["graphene", "graphql-server<3.0.0b8"]},
```

Comment:
```suggestion
    # We place an upper bound on graphql-server because it has undergone a major
    # change in 3.0.0b8 in Aug 2025. When graphql-server 3 has stabilized we'll need
    # to update this module to make it compatible with it.
    "external_dependencies": {"python": ["graphene", "graphql-server<3.0.0b8"]},
```